torch/csrc/cuda/Module.cpp

```diff
@@ -596,186 +595,14 @@ PyObject* THCPModule_resetPeakMemoryStats(PyObject* _unused, PyObject* arg) {
   Py_RETURN_NONE;
 }
 
-struct Frame {
```

Comment:
This code is moving to `combined_traceback.cpp`. The change in logic is only to: * remove pybind11 specific code, and * deduplicate repeat Python stack frames to avoid creating additional python objects.